frontend/express/public/core/user-management/javascripts/countly.views.js

```diff
@@ -0,0 +1,261 @@
+/*global countlyAuth, app, countlyGlobal, CV, countlyVue, countlyUserManagement, countlyCommon, CountlyHelpers */
+(function() {
+    var FEATURE_NAME = "global_users";
+
+    var DataTable = countlyVue.views.create({
+        template: CV.T("/core/user-management/templates/data-table.html"),
+        mixins: [countlyVue.mixins.commonFormatters],
+        props: {
+            rows: Array
+        },
+        data: function() {
+            return {};
+        },
+        created: function() {
+        },
+        methods: {
+            handleCommand: function(command, index) {
+                switch (command) {
+                case "delete-user":
+                    // TODO: wrap it with popup
+                    countlyUserManagement.deleteUser(index, function() {
+                        // TODO: show toast
+                    });
+                    break;
+                case 'edit-user':
+                    this.$emit('edit-user', index);
+                    break;
+                }
+            }
+        }
+    });
+
+    var Drawer = countlyVue.views.create({
+        template: CV.T("/core/user-management/templates/drawer.html"),
+        props: ['settings', 'controls', 'features'],
```

Comment:
It would be better being more specific about prop types, default values, etc. (e.g. rows: Array)